### PR TITLE
Added support for battery life on os x

### PR DESCRIPTION
**Pull request by zzZIMAWAKE**
Added support for battery life on os x, haven't been able to test this on linux but shouldn't effect it.

### CODE REVIEW

**Comment by bre7** [comment]
You can get battery level (as %, no need for conversions) with pmset also.

Battery level:

```
pmset -g batt | grep -o '[0-9]*%' | tr -d %
```

Is charging:

```
pmset -g batt | grep -c 'Battery Power'
```

**Comment by jose-lpa** [comment]
Great job, thanks. Works perfectly in Yosemite, FYI. Would be nice to see this PR merged.

File: agnosterzak.zsh-theme [hunk 1]
```diff
@@ -89,6 +89,53 @@ prompt_context() {
 prompt_battery() {
   HEART='♥ '
 
+  if [[ "$OSTYPE" = darwin* ]] ; then
+
```
Comment (jose-lpa) [inline agnosterzak.zsh-theme:93]: Just a very minor appreciation: `if [[ $(uname) == "Darwin"  ]] ; then` works as well, and looks similar than the basecode line for Linux systems :)

**Review by jose-lpa — APPROVED** [review]

**Comment by zzZIMAWAKE** [comment]
@jose-lpa Thanks, I've updated with your suggestion. Not sure @zakaziko99 is all that active on this repo, not sure it will get merged any time soon as it's sat here since March. Hopefully he'll get round to it at some point

**Comment by NaifAlrashed** [comment]
thanks! I had to copy and past code manually, but it works on my Sierra

**Comment by Piorek99** [comment]
@NaifAlrashed can you upload a file with battery file for macos?

**Pull request merged**